### PR TITLE
Add multi-container test API

### PR DESCRIPTION
**Pull request by RobertLucian**
checklist:

- [x] run `make test` and `make lint`
- [x] test manually
- [x] push images to `quay.io/cortexlabs-test`.

### CODE REVIEW

File: test/apis/realtime/multi-container/app/main.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+
+@app.post("/")
+def text_generator(request: Request):
+    server_url = f"http://localhost:8501/v1/models/resnet50:predict"
+
+    # download labels
+    labels = requests.get(
+        "https://storage.googleapis.com/download.tensorflow.org/data/ImageNetLabels.txt"
+    ).text.split("\n")[1:]
```
Comment (deliahu) [inline test/apis/realtime/multi-container/app/main.py:27]: Should these be moved outside of the `text_generator()` method?

File: test/apis/realtime/multi-container/build-cpu-tfs.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# usage: build-cpu.sh [REGISTRY] [--skip-push]
+#   REGISTRY defaults to $CORTEX_DEV_DEFAULT_IMAGE_REGISTRY; e.g. 764403040460.dkr.ecr.us-west-2.amazonaws.com/cortexlabs or quay.io/cortexlabs-test
+
+image_name="realtime-multi-container-cpu-tfs"
```
Comment (deliahu) [inline test/apis/realtime/multi-container/build-cpu-tfs.sh:6]: nit: I think I'd switch it so the file was named `build-tfs-cpu.sh` and the image was named `realtime-multi-container-tfs-cpu`, (also for web), what do you think?

File: test/apis/realtime/multi-container/app/main.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    dl_request.raise_for_status()
+
+    # compose a JSON Predict request (send JPEG image in base64).
+    jpeg_bytes = base64.b64encode(dl_request.content).decode("utf-8")
+    predict_request = '{"instances" : [{"b64": "%s"}]}' % jpeg_bytes
+
+    # make prediction
+    response = requests.post(server_url, data=predict_request)
+    response.raise_for_status()
+    return {"image_prediction": labels[response.json()["predictions"][0]["classes"]]}
```
Comment (deliahu) [inline test/apis/realtime/multi-container/app/main.py:40]: nit: split this into 2 lines, e.g.

```python
label_id = response.json()["predictions"][0]["classes"]
return {"image_prediction": labels[label_id]}
```

File: test/apis/realtime/multi-container/app/main.py (hunk 1 above)
Comment (RobertLucian) [reply test/apis/realtime/multi-container/app/main.py:27]: Yep, they can. Same can be said about `server_url`.

**Review by deliahu — APPROVED** [review]

**Pull request merged**